dev/tools/dartdoc.dart

```diff
@@ -355,7 +355,6 @@ void sanityCheckDocs() {
     '$kPublishRoot/api/dart-io/File-class.html',
     '$kPublishRoot/api/dart-ui/Canvas-class.html',
     '$kPublishRoot/api/dart-ui/Canvas/drawRect.html',
-    '$kPublishRoot/api/flutter_driver/FlutterDriver/FlutterDriver.connectedTo.html',
```

Comment:
Did this change automatically?

Reply:
I manually deleted this. The base FlutterDriver class no longer has connectedTo method.